form/form.go

```diff
@@ -454,12 +442,6 @@ func parseTag(tag string) (string, *formOptions) {
 			}
 			options.Empty = true
 
-		case "indexed":
```

Comment:
Removed the `indexed` option because everything is indexed now.

Reply:
Flagging that we likely could/should remove all options. The only one really used is `empty` specifically for additional_owners I think.

Reply:
+1. Let's do that as a separate cleanup, but yep, we should be able to kill all of them at this point.